eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/policies/SkipPolicyVerification.java

```diff
@@ -0,0 +1,21 @@
+package uk.ac.ebi.eva.accession.pipeline.policies;
+
+import org.springframework.batch.core.step.skip.SkipLimitExceededException;
+import org.springframework.batch.core.step.skip.SkipPolicy;
+import org.springframework.batch.item.file.FlatFileParseException;
+
+import uk.ac.ebi.eva.commons.core.models.factories.exception.IncompleteInformationException;
+import uk.ac.ebi.eva.commons.core.models.factories.exception.NonVariantException;
+
+public class SkipPolicyVerification implements SkipPolicy {
```

Comment:
This class name is not very descriptive of what is skipped. We also tend to use the class names linked to the hierarchy at the end of the class names. What about `NonVariantsSkipPolicy`? Some class documentation would be also useful.